src/terms/xc.jl

```diff
@@ -82,46 +85,49 @@ end
     tσ = libxc_spinindex_σ
 
     # Potential contributions Vρ -2 ∇⋅(Vσ ∇ρ) + ΔVl
-    potential = zero(ρ)
-    @views for s in 1:n_spin
-        potential[:, :, :, s] .+= terms.vrho[s, :, :, :]
-        if haskey(terms, :vsigma) && any(x -> abs(x) > term.potential_threshold, terms.vsigma)
-            # Need gradient correction
-            # TODO Drop do-block syntax here?
-            potential[:, :, :, s] .+= -2divergence_real(basis) do α
-                # Extra factor (1/2) for s != t is needed because libxc only keeps σ_{αβ}
-                # in the energy expression. See comment block below on spin-polarised XC.
-                sum((s == t ? one(T) : one(T)/2)
-                    .* terms.vsigma[tσ(s, t), :, :, :] .* density.∇ρ_real[t, :, :, :, α]
-                    for t in 1:n_spin)
-            end
-        end
-        if haskey(terms, :vlapl) && any(x -> abs(x) > term.potential_threshold, terms.vlapl)
-            @warn "Meta-GGAs with a Vlapl term have not yet been thoroughly tested." maxlog=1
-            mG² = [-sum(abs2, G) for G in G_vectors_cart(basis)]
-            Vl_fourier = r_to_G(basis, terms.vlapl[s, :, :, :])
-            potential[:, :, :, s] .+= G_to_r(basis, mG² .* Vl_fourier)  # ΔVl
-        end
-    end
+    potential = vrho #zero(ρ)
```

Comment:
that should work as well, but do we even need the permutation if we can just slice the arrays differently in the function calls?

Reply:
I'm not sure what you mean. My point for suggesting this was to stay as close to the uncommented code as possible. But with the `map` approach I suggested below this will be obsolete anyway. Let's talk tomorrow.